openedx/core/djangoapps/user_api/accounts/image_helpers.py

```diff
@@ -37,21 +38,20 @@ def _make_profile_image_name(username):
     return hashlib.md5(settings.PROFILE_IMAGE_SECRET_KEY + username).hexdigest()
 
 
-def _get_profile_image_filename(name, size):
+def _get_profile_image_filename(name, size, file_extension='jpg'):
     """
     Returns the full filename for a profile image, given the name and size.
     """
-    return '{name}_{size}.jpg'.format(name=name, size=size)
+    return '{name}_{size}.{file_extension}'.format(name=name, size=size, file_extension=file_extension)
 
 
-def _get_profile_image_urls(name):
+def _get_profile_image_urls(name, storage, file_extension='jpg'):
```

Comment:
Good point. I'll make this a constant. I considered converting the default images to JPEG, but all of our other static assets are PNG. It also seemed useful to allow the Open edX community to be able to make static images in whatever format they desire. We chose JPEG because it works best for photos, but the static images are likely not photos so it didn't seem like a restriction I wanted to make.